packages/manager/apps/web/client/app/hosting/database-order/database-order.routing.js

```diff
@@ -0,0 +1,38 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.database-order', {
+    url: '/database-order',
+    views: {
+      '@app.hosting': {
+        component: 'hostingDatabaseOrderComponent',
+      },
+    },
+    resolve: {
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('hosting_database_order_breadcrumb'),
+      getHostingsIceberg: /* @ngInject */ (iceberg) =>
+        iceberg('/hosting/web')
+          .query()
+          .expand('CachedObjectList-Pages')
+          .sort('displayName', 'ASC')
+          .execute()
+          .$promise.then(({ data }) =>
+            data.map((item) => ({
+              ...item,
+              displayName: item.displayName || item.serviceName,
+              serviceName: item.serviceName,
```

Comment:
is that necessary ?

Reply:
I mean the last line `serviceName: item.serviceName` is useless I think, since it is already in `item`